src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -29,7 +29,7 @@ const CLUSTER_NAME_START_END_REGEX = /^[a-z0-9](.*[a-z0-9])?$/;
 const CLUSTER_NAME_VALID_CHARS_REGEX = /^[a-z0-9-]*$/;
 const SSH_PUBLIC_KEY_REGEX =
   /^(ssh-rsa|ssh-ed25519|ecdsa-[-a-z0-9]*) AAAA[0-9A-Za-z+/]+[=]{0,3}( .+)?$/;
-const DNS_NAME_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/;
+const DNS_NAME_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*.)+[a-z]{2,}$/;
```

Comment:
Let's see how it works in CIM, but maybe we need to have separate validations as it can have a large impact.

Also, the regexp should probably be 
```
const DNS_NAME_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*)+[a-z]{2,}$/;
```
as you removed the scape character for the "dot", but not the "dot" itself (which without scaping, then it means any character, and we already have the asterisk for that).

I haven't tested it, I'm saying this from a theoretical POV.

Reply:
After looking at the screenshot on the ticket about the validation error, I think the RegExp should be much more relaxed, as the last part is about the domain part (.co), so if there can be no domain we may not enforce anything.

Maybe we also need to check what the regexp is in the backend and do the same.

Reply:
I've changed to use the same validation that BE: 
https://github.com/openshift/assisted-service/blob/master/pkg/validations/validations.go#L17

const DNS_NAME_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*)+$/;